packages/realm/src/app-services/SyncConfiguration.ts

```diff
@@ -69,10 +69,16 @@ export enum SessionStopPolicy {
   Never = "never",
 }
 
+/**
+ */
 export enum ClientResetMode {
+  /** @deprecated See {@link Realm.App.Sync.initiateClientReset} */
   Manual = "manual",
+  /** Download a fresh copy from the server. */
```

Comment:
Any reason why these are comment blocks?  In any case, it, and others, should adhere to the same style as the majority of them.
```suggestion
  /** 
   * Download a fresh copy from the server.
   */
```

Reply:
Because they are tsdoc comments (notice the two stars instead of one on the first line), they cannot be single line comments (`//`) as that won't be picked up by tsdoc parsers.

Reply:
Then we should align on what a single line comment should be.  Either:
```
/**
 *
 */
 ```
or
```
/**  */
```
and I've also seen
```
/* */
```
Seems the first is most common